pkg/aws/disk.py

```diff
@@ -7,27 +7,39 @@
 from asyncio.subprocess import PIPE, STDOUT 
 from model.discover import *
 from model.project import *
+from ansible_runner import run_async
 
-async def start_cloning(project, hostname):
+async def start_cloning(project):
     con = create_db_con()
     try:
         bucket = Bucket.objects(project=project).allow_filtering()[0]['bucket']
         accesskey = Bucket.objects(project=project).allow_filtering()[0]['access_key']
         secret_key = Bucket.objects(project=project).allow_filtering()[0]['secret_key']
-        public_ip = Discover.objects(project=project,host=hostname).allow_filtering()[0]['public_ip']
+        provider = Project.objects(name=project).allow_filtering()[0]['provider']
```

Comment:
this will start cloning for all the hosts in the project. We should be also able to do cloning one by one